mod/deckorate_search/set/self/search/open_search.rb

```diff
@@ -92,4 +92,12 @@ def os_type_filter
 
     yield[:filter] = { term: { type_id: type_param.card_id } }
   end
+
+  def suggest_contexts
+    return filter_type_ids unless filter_type_ids.length() > 1
+    contexts = []
+    filter_type_ids.each { |card_id| contexts.append({ "context": card_id, "boost": 2 }) unless card_id == 629 }
+    contexts.append({ "context": 629 })
+    return contexts
```

Comment:
I can't really read this code and understand what it's doing.  Why would you add all of them but 629 and then add 629?  Is it an order thing?

Refactoring might fix it, but otherwise we'd probably need a comment.